firefox_desktop/explores/newtab_interactions.explore.lkml

```diff
@@ -19,4 +19,174 @@ explore: newtab_interactions {
     relationship: one_to_one
     sql_on: ${newtab_interactions.country_code} = ${countries.code} ;;
   }
+
+  aggregate_table: rollup__newtab_interactions_general_per_day {
+    query: {
+      dimensions: [
+        newtab_interactions.submission_date,
+        newtab_interactions.channel,
+        newtab_interactions.country_code,
+        newtab_interactions.activity_segment,
+        newtab_interactions.pocket_enabled,
+        newtab_interactions.pocket_is_signed_in,
+        newtab_interactions.pocket_sponsored_stories_enabled,
+        newtab_interactions.topsites_enabled,
+        newtab_interactions.topsites_rows,
+        countries.tier,
+        countries.name,
+        countries.pocket_available_on_newtab,
+      ]
+      measures: [
+        newtab_interactions.visits,
+        newtab_interactions.clients,
```

Comment:
I'll keep an eye on this. The documentation [here](https://cloud.google.com/looker/docs/aggregate_awareness#approximate_distinct) mentions that distinct counts are only pre-aggregated if all the dimensions are included but an exception is approximate distincts. These are all approximate distincts but the documentation is unclear on whether this will be pre-aggregated in an example say only using one or two of these dimensions.

Reply:
This doesn't seem to be working, will follow up in https://mozilla-hub.atlassian.net/browse/DS-2592